composer.json

```diff
@@ -16,7 +16,7 @@
     ],
     "require": {
         "php": ">=7.1.0",
-        "laravel/tinker": "^1.0",
+        "laravel/tinker": "1 - 2",
```

Comment:
This looks like it's the "wrong" way to handle this from what I've seen.
pretty sure it's supposed to be `"^1.0|^2.0"`based on other repos requiring this same fix.